org.sonarlint.eclipse.ui/src/org/sonarlint/eclipse/ui/internal/views/locations/IssueLocationsView.java

```diff
@@ -235,10 +235,8 @@ public IMarker getMarker() {
 
             p -> (ExtraPosition) p)
           .filter(filter).collect(Collectors.toList());
-      } catch (
-
-      BadPositionCategoryException e) {
-        SonarLintLogger.get().error("Unable to read positions", e);
+      } catch (BadPositionCategoryException e) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1166)